input/pagecontent/reference.md

```diff
@@ -1682,13 +1700,23 @@ Creating a code system uses the keywords `CodeSystem`, `Id`, `Title` and `Descri
 * #{code} "{display string}" "{definition string}"
 ```
 
+Codes that are contained within other codes can also be defined. The result is a hierarchical structure of codes within a code system. To define such codes, list all of the preceding codes in the hierarchy before the new code:
+
+
+```
+* #{existing code} #{code} "{display string}" "{definition string}"
```

Comment:
To get the point across that it can be more than one code deep, I'd suggest maybe
```suggestion
* #{parent code} #{code} "{display string}" "{definition string}"
* #{grandparent code} #{parent code} #{code} "{display string}" "{definition string}"
```